website/newsletters/templates/newsletters/email.html

```diff
@@ -93,7 +93,7 @@ <h2 style="{{ item_h2_style }}">{{ item.title.upper }}</h2>
                 {% if item.newsletterevent %}
                 <tr>
                     <td style="{{ item_tr_style }}">
-                        {{ item.description|safe }}
+                        {{ item.short|safe }}
```

Comment:
This `item.description` refers to the newsletter description. If you want to fill in the short description from the event in the newsletter, you will need to change it in the javascript here: https://github.com/svthalia/concrexit/blob/507a901d9c3d1b669933c275cafe4b806bba9be5/website/newsletters/static/admin/newsletters/js/forms.js#L28